### ISSUE TITLE
Can Cross Entropy Loss Be Robust to Label Noise?

### ISSUE THREAD

**Issue by riron1206** [riron1206/read_paper#46]
## 概要
https://www.ijcai.org/Proceedings/2020/305
ラベルノイズにロバストなTaylor Cross Entropy (TCE) loss

TCEはテイラー級数展開でCCEの効果を緩めたCategorical Cross Entropy (CCE)
MAE はCCEの1次テイラー級数近似（TCEのt=1）
MAE と平均2乗誤差(MSE)の平均がCCEの2次テイラー級数近似（TCEのt=2）
次数大きなTCEはCCEに近づく

MAEは単純な差だから劇的な性能低下はないが、
CCEはラベルノイズ下では性能劇的に低下（外れ値に敏感のこと？シグモイドだからラベル間違ってるサンプルの効果強く受けるから？）

MAE組み込んだ他のloss
- Generalized Cross Entropy (GCE) loss: MAEとCCEのバランスをとるためのハイパーパラメータqを使うloss
- Partial Huberised Cross Entropy (PHuberCE) loss: 勾配クリッピングでCCEを補正するloss
- Symmetric Cross Entropy (SCE) loss: 正則化パラメータを調整してCCEとMAEを組み合わせたloss

バイテンパーロス（外れ値のロスを減らす+softmaxの裾の幅広げてラベルノイズの効果弱めるloss） 参考< https://github.com/riron1206/read_paper/issues/42 >
とは精度比較してないのは作為的なものを感じる
パラメータは次数tだけなのはいいけど

![image](https://user-images.githubusercontent.com/22429441/104732353-724cd380-5780-11eb-8760-a5d494d229f6.png)

## 結果
t=2でラベルノイズの10%のMNISTやCIFAR-10でSCEなどより高精度

## コメント
- 非公式のgithub: 
    - https://github.com/CoinCheung/pytorch-loss
